granule_ingester/granule_ingester/writers/SolrStore.py

```diff
@@ -184,7 +184,8 @@ def _build_solr_doc(self, tile: NexusTile) -> Dict:
             'tile_max_lon': bbox.lon_max,
             'tile_min_lat': bbox.lat_min,
             'tile_max_lat': bbox.lat_max,
-            'tile_depth': tile_data.depth,
+            'tile_min_elevation': tile_data.min_elevation,
+            'tile_max_elevation': tile_data.max_elevation,
```

Comment:
@RKuttruff By naming the solr fields tile_min_elevation and tile_max_elevation, would we need to explicitly define them in https://github.com/apache/incubator-sdap-nexus/blob/master/docker/solr/cloud-init/create-collection.py#L140-L144 ? Or should we use dynamically typed fields e.g. tile_min_elevation_d and tile_max_elevation_d

Reply:
I think adding it to the script will be the easier route since it won't require altering all the queries, etc
I already added this to the nexus PR

Reply:
One potential problem is that for existing SDAP deployments the create-collection script isn't ran. If we use dynamically typed fields then existing deployments don't need to manually define these fields.

Reply:
I'll do both then

Reply:
@ngachung done